getting_started/first_3d_game/09.adding_animations.rst

```diff
@@ -259,16 +261,22 @@ node structure, you can copy them to different scenes.
 For example, both the ``Mob`` and the ``Player`` scenes have a ``Pivot`` and a
 ``Character`` node, so we can reuse animations between them.
 
-Open the *Player* scene, select the AnimationPlayer node and open the "float"
-animation. Next, click on **Animation > Copy**. Then open ``mob.tscn``,
-create an AnimationPlayer child node and select it. Click **Animation > Paste**
-and make sure that the button with an "A+" icon (Autoplay on Load) and the
-looping arrows (Animation looping) are also turned on in the animation editor
-in the bottom panel. That's it; all monsters will now play the float animation.
+Open the *Player* scene, select the AnimationPlayer node and then click on
+**Animation > Manage Animations...**. Click the *Copy animation to clipboard* button
+(two small squares) alongside the *float* animation. Click OK to close the window.
+
+Then open ``mob.tscn``, create an :ref:`AnimationPlayer <class_AnimationPlayer>` child
+node and select it. Click **Animation > Manage Animations**, then **Add Library**. You
+should see the message "Global library will be created." Leave the text field blank and
+click OK. Click the *Paste* icon (clipboard) and it should appear in the window. Click OK
+to close the window.
+
+Next, make sure that the button with an "A+" icon (Autoplay on Load) and the looping
+arrows (Animation looping) are also turned on in the animation editor in the bottom panel.
+That's it; all monsters will now play the float animation.
```

Comment:
The icon isn't "A+", it's more like "A>":

![image](https://github.com/godotengine/godot-docs/assets/180032/fcecaf60-05e6-4a60-a92f-c97c818e1208)

That said, I feel writing the icon as text isn't ideal as it can end up being confusing.

We should probably look into introducing inline icons in text in a future PR.

Reply:
Yes, I noticed this. It was written as A+ in other parts of the tutorials so I just copied that.

I can change this (and all the other occurrences that were already there) if that's the path forward.